pype/plugins/ftrack/publish/integrate_hierarchy_ftrack.py

```diff
@@ -35,7 +35,7 @@ class IntegrateHierarchyToFtrack(pyblish.api.ContextPlugin):
 
     order = pyblish.api.IntegratorOrder - 0.04
     label = 'Integrate Hierarchy To Ftrack'
-    families = ["clip", "shot"]
+    families = ["shot"]
```

Comment:
are we 100% certain that this will not affect publishing from hiero?

Reply:
I  believe Hiero should be fine, but Premiere is effected by this change..